src/dev/typescript/projects.ts

```diff
@@ -31,3 +31,12 @@ export const PROJECTS = [
   // both took closer to 1000ms.
   ...glob.sync('packages/*/tsconfig.json', { cwd: REPO_ROOT }),
 ].map(path => new Project(resolve(REPO_ROOT, path)));
+
+export function filterProjectsByFlag(projectFlag: string | undefined) {
+  if (!projectFlag) {
+    return PROJECTS;
+  }
+
+  const tsConfigPath = resolve(projectFlag);
+  return PROJECTS.filter(project => project.tsConfigPath === tsConfigPath);
```

Comment:
__question:__ isn't it just `find`? Or it's just a trick to not do `if (.... === null) return []`?

Reply:
> but filter gets us there in one step.
and always iterates through entire array, but yeah it's not an issue here :)